webapp/src/components/modals/actionModal.tsx

```diff
@@ -0,0 +1,173 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+
+import './styles.css';
+
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
```

Comment:
Group it with the other react-bootstrap imports

Reply:
I think you can add all the other imports in this import only. No need to import them separately.

Reply:
I have tried importing all the bootstrap component in this import only. But it is giving random errors. This was the fix suggested by internet

Reply:
Group the imports